yahoo_fantasy_api/league.py

```diff
@@ -284,3 +294,56 @@ def _date_range_of_played_or_current_week(self, week):
                 datetime.datetime.strptime(j['week_start'], "%Y-%m-%d").date(),
                 datetime.datetime.strptime(j['week_end'], "%Y-%m-%d").date())
         return self.week_date_range_cache[week]
+
+    def player_details(self, player_name):
+        """Retrieve details about a specific player
+        
+        :parm player_name: Name of the player to get details for
+        :type player_name: str
+        :return: Player details
+        :rtype: dict
+        """
+        """Retrieve details about a specific player
+        
+        :parm player_name: Name of the player to get details for
+        :type player_name: str
+        :return: Player details
+        :rtype: dict
+        >>> lg.player_details("Antonio Brown")
```

Comment:
I like the extra detail to show what the output looks like.  But part of this docstring is duplicated above.